src/components/Footer.tsx

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import { Layout, Row, Col, Typography, Space, Button, Image } from "antd";
+import {
+  GithubOutlined,
+  TwitterOutlined,
+  MailOutlined,
+  LinkedinOutlined,
+} from "@ant-design/icons";
+
+const { Footer } = Layout;
+const { Text, Link } = Typography;
+
+const footerSections = [
```

Comment:
Can we please move this outside to a `constants/content/<component>.json` folder?

Reply:
yeah. had this in mind! plan on doing this. restructuring the whole project folder structure in  another task.

would def resolve here.